consumers/src/unit-test/java/uk/gov/moj/sdt/consumers/client/impl/ClaimStatusUpdateRequestServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.moj.sdt.consumers.client.impl;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.moj.sdt.cmc.consumers.api.CMCApi;
+import uk.gov.moj.sdt.cmc.consumers.client.impl.ClaimStatusUpdateService;
+import uk.gov.moj.sdt.cmc.consumers.request.ClaimStatusUpdateRequest;
+import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
+
+
+@ExtendWith(MockitoExtension.class)
+public class ClaimStatusUpdateRequestServiceTest extends AbstractSdtUnitTestBase {
```

Comment:
The tests in this class need to be finished.

Reply:
deleted this